### PR TITLE
Fixed sound, and clean up

### PR DESCRIPTION
**Pull request by tinkerer-red**
The sound files were not being set properly with function spawn.
some commands would post messages twice when only one was required.
aligned the suggested commands in order of which a player would likely use them

### CODE REVIEW

File: main.lua [hunk 1]
```diff
@@ -221,6 +221,11 @@ function DoorFramework.cmdRecord(pid, cmd)
                 messageCmdUsage(pid, "Model path is not set!\n")
                 return
             end
+            
+            if param.sound == nil then
+                param.sound = DoorFramework.config.sound
+                messageCmdUsage(pid, "Model path is not set!\n")
```
Comment (uramer) [inline main.lua:227]: Type in the message

**Review by uramer — CHANGES_REQUESTED** [review]
Fix typo in sound message, and I will approve the PR

Thank you!

**Comment by tinkerer-red** [comment]
dh doesn't work, i believe they updated it?

**Comment by uramer** [comment]
Would be nice if you removed the todo, as this is my repo and I personally don't plan to do that in the near future

**Comment by tinkerer-red** [comment]
You got it, sorry I thought I was only requesting patch to the lua.

**Review by uramer — APPROVED** [review]

**Pull request merged**